src-browser/perf_hooks.js

```diff
@@ -0,0 +1,31 @@
+function unimplemented() {
+  throw new Error(
+    "Node.js performance is not supported by JSPM core in the browser"
+  );
+}
```

Comment:
Let's rather make this message function specific:

```suggestion
function unimplemented (functionName) {
  throw new Error(
    "Node.js performance method ${functionName} is not currently supported by JSPM core in the browser"
  );
}
```

Reply:
I actually felt the same, the above one looked too generic and confusing while some methods do work 😅